.husky/commit-msg

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
this is cool - perhaps an implication is that if you install `git` without updating `PATH` you might get errors running shell commands from the hook.

we may already have these linux command line tools as a requirement for our repo

Reply:
Husky claims to work on Windows as long as Git is installed: https://typicode.github.io/husky/#/?id=does-it-work-on-windows

Since Git hooks written in Bash are a feature built into Git itself, and Husky merely automates creating/managing these hooks, I don't think it's necessary for Bash to be on the PATH as long as it's present in the Git installation. I should probably test this in a Windows Sandbox to be sure 😛

Reply:
cool beans 😎

Reply:
thank you 😄